install.py

```
@@ -0,0 +1,349 @@
+        print(f"   Bitte besuchen Sie: {ollama_url}")
+        print("   Oder führen Sie aus: winget install Ollama.Ollama")
+        
+    elif system == "darwin":
+        print("📥 Installiere Ollama für macOS...")
+        print("   Führen Sie aus: brew install ollama")
+        
+    elif system == "linux":
+        print("📥 Installiere Ollama für Linux...")
+        try:
+            # Lade Ollama-Installationsskript
+            subprocess.check_call([
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)